src/Donors/Repositories/DonorRepository.php

```diff
@@ -505,6 +506,12 @@ private function getWhereConditionsForRequest(QueryBuilder $builder, WP_REST_Req
             $builder->where('date_created', $end, '<=');
         }
 
+        if ($form) {
+            $builder->whereLike('payment_ids', $form . ',%');
```

Comment:
Huh, I think we should update QB to help us handle situations like this. We could probably use regex to match the exact phrase, so adding something like `whereRegex` to QB might be useful. Performance-wise, I think it should be similar to using `LIKE` operator.

Reply:
Oooh, good idea!